eng/Versions.props

```diff
@@ -25,23 +25,23 @@
     <MicrosoftVisualStudioExtensibilityTestingVersion>0.1.149-beta</MicrosoftVisualStudioExtensibilityTestingVersion>
     <!-- CodeStyleAnalyzerVersion should we updated together with version of dotnet-format in dotnet-tools.json -->
     <CodeStyleAnalyzerVersion>4.3.0-1.final</CodeStyleAnalyzerVersion>
-    <VisualStudioEditorPackagesVersion>17.5.42-preview</VisualStudioEditorPackagesVersion>
+    <VisualStudioEditorPackagesVersion>17.5.116-preview</VisualStudioEditorPackagesVersion>
```

Comment:
this PR will go into main-vs-deps.

Reply:
do we actually need main-vs-deps?  If these are in 17.5p1 (which is released) I don't think we need it.  If they are in preview 2 then we do

Reply:
these are not in p1.  we need main-vs-deps here afaict.

Reply:
Per the internal schedule, 17.5 Preview 1 was only sent over to the scouting queue today anyways, so that wouldn't have been available yet.

Reply:
@jasonmalinowski so do i need to change something here?

Reply:
I think you're good, as long as we're not dropping integration tests on the floor with this.